etc/picongpu/dicc-um/gpu_picongpu.profile.example

```diff
@@ -0,0 +1,49 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ################################# (edit the following lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="END"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# basic environment ###########################################################
+#
+module load git
+module load openpmd-api/openpmd-api-0.14.4-gcc-8.5.0
+module load boost/boost-1.74.0-gcc-8.5.0
+
+export CC=$(which gcc)
+export CXX=$(which g++)
+
+# plugins (optional) ##########################################################
+module load pngwriter/pngwriter-0.7.0-gcc-8.5.0
+
+export PICSRC=$HOME/src/picongpu
+export PIC_EXAMPLES=$PICSRC/share/picongpu/examples
+export PIC_BACKEND="cuda:30"  
+export PATH=$PICSRC:$PATH
+export PATH=$PICSRC/bin:$PATH
+export PATH=$PICSRC/src/tools/bin:$PATH
+export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH
+
+export SCRATCH=/lustre/user/$(whoami)
+
+export TBG_SUBMIT="sbatch"
+export TBG_TPLFILE="$HOME/gpu.tpl"
```

Comment:
This is non standard location. Please change to `etc/picongpu/dicc-um/gpu.tpl`

Reply:
Done